packages/app-bridge/src/AppBridgePlatformApp.ts

```diff
@@ -94,35 +98,45 @@ export class AppBridgePlatformApp implements IAppBridgePlatformApp {
     api<ApiMethodName extends keyof PlatformAppApiMethod>(
         apiHandler: ApiHandlerParameter<ApiMethodName, PlatformAppApiMethod>,
     ): ApiReturn<ApiMethodName, PlatformAppApiMethod> {
-        return this.messageBus.post({
-            method: 'api',
+        return this.apiMessageBus.post({
             parameter: apiHandler,
         }) as ApiReturn<ApiMethodName, PlatformAppApiMethod>;
     }
 
+    private initializationGuard() {
+        const initialContext = getQueryParameters(window.location.href);
+        if (!initialContext.token) {
+            throw new InitializationError();
+        }
+        if (this.initialized) {
```

Comment:
What if `this.initialized` evaluates falsy? We would still return `undefined`. Should it also throw then?

Reply:
When developing an app you have the strict mode that rerenders twice the apps. If that happens the object is already initiated and doesnt need another initiation (It already has the needed objects from the web-app). It can just be ignored and nothing needs to happen.